### ISSUE TITLE
cocoa pods fails to intall

### ISSUE THREAD

**Issue by romk1n** [viki-org/VKVideoPlayer#58]
```shell
Updating local specs repositories
Analyzing dependencies
Downloading dependencies
Installing AFNetworking (1.3.3)
Installing CocoaAsyncSocket (7.4.2)
Installing CocoaHTTPServer (2.3)
Installing CocoaLumberjack (1.7.0)
Installing DTCoreText (1.6.16)
Installing DTFoundation (1.7.8)
Installing Expecta (0.3.2)
Installing FXImageView (1.3.5)
Installing OCMock (2.2.4)
Installing Reachability (3.1.1)
Installing Specta (0.2.1)
Installing VKFoundation (0.1.1)
Generating Pods project
[!] An error occurred while processing the post-install hook of the Podfile.

undefined method `project' for #<Pod::Installer:0x007f9078cb7d88>

/Users/roman/devel/github/VKVideoPlayer/Podfile:17:in `block (2 levels) in from_ruby'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-core-0.39.0/lib/cocoapods-core/podfile.rb:170:in `call'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-core-0.39.0/lib/cocoapods-core/podfile.rb:170:in `post_install!'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:806:in `run_podfile_post_install_hook'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:794:in `block in run_podfile_post_install_hooks'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:140:in `message'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:793:in `run_podfile_post_install_hooks'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:158:in `block in generate_pods_project'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:153:in `generate_pods_project'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:111:in `install!'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:101:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
```

**Comment by ftvs** [comment]
Change project to pods_project.

https://github.com/viki-org/VKVideoPlayer/issues/47
https://github.com/CocoaPods/CocoaPods/issues/3747

**Comment by Shannon-Yang** [comment]
I also encountered the same problem, but I have resolved, the solution here https://github.com/CocoaPods/CocoaPods/issues/3918

**Comment by liujieyuu** [comment]
i have a problem at pod install: 
 - Running post install hooks
[!] An error occurred while processing the post-install hook of the Podfile.

undefined method `VKVideoPlayer' for #<Pod::Installer:0x007feea925d8f8>

/Users/pastjie/Desktop/VKVideoPlayer/Podfile:24:in `block (2 levels) in from_ruby'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-core-0.39.0/lib/cocoapods-core/podfile.rb:170:in `call'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-core-0.39.0/lib/cocoapods-core/podfile.rb:170:in `post_install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:806:in `run_podfile_post_install_hook'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:794:in `block in run_podfile_post_install_hooks'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:140:in `message'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:793:in `run_podfile_post_install_hooks'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:158:in `block in generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:153:in `generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:111:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:101:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'

/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:808:in `rescue in run_podfile_post_install_hook'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:806:in `run_podfile_post_install_hook'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:794:in `block in run_podfile_post_install_hooks'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:140:in `message'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:793:in `run_podfile_post_install_hooks'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:158:in `block in generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:153:in `generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:111:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:101:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'

**Comment by sahin** [comment]
I suggest everyone to check out https://github.com/mobileplayer/mobileplayer-ios

**Comment by luugiathuy** [comment]
Fixed in #59
